### PR TITLE
fix: refresh table info after zk updated

### PR DESCRIPTION
**Pull request by vagetablechicken**
cicd workflow cpp job failed on deploy, cuz tablet may overwrite index by old zk info(slow notify, and no version, so it replace new), so make tablet sync zk info after zk info update.

### CODE REVIEW

File: src/nameserver/name_server_impl.cc [hunk 1]
```diff
@@ -8584,6 +8582,22 @@ bool NameServerImpl::AddIndexToTableInfo(const std::string& name, const std::str
         }
     }
     UpdateZkTableNode(table_info);
+    // refresh tablet here, cuz this func may be called by task
```
Comment (dl239) [inline src/nameserver/name_server_impl.cc:8585]: get all tablet by `GetAllHealthTablet` and call refresh

Comment (vagetablechicken) [reply src/nameserver/name_server_impl.cc:8585]: tid is just one, I'll fix it, and filter tablet by health and exists in table info. GetAllHealthTablet has a lock in it, so just filter health in here

File: src/test/util.cc [hunk 2]
```diff
@@ -182,15 +182,15 @@ void ProcessSQLs(sdk::SQLRouter* sr, std::initializer_list<absl::string_view> sq
     hybridse::sdk::Status status;
     for (auto sql : sqls) {
         sr->ExecuteSQL(std::string(sql), &status);
-        EXPECT_TRUE(status.IsOK()) << "running sql=" << sql << " failed: code=" << status.code << ", msg=" << status.msg
+        ASSERT_TRUE(status.IsOK()) << "running sql=" << sql << " failed: code=" << status.code << ", msg=" << status.msg
```
Comment (aceforeverd) [inline src/test/util.cc]: we need `EXPECT*`, some context require all SQLs executed even failure

**Review by aceforeverd — CHANGES_REQUESTED** [review]

File: src/test/util.cc (hunk 2 above)
Comment (vagetablechicken) [reply src/test/util.cc]: OK, but it's not a good idea to expect when init(e.g. create db/table, deploy...), if create failed, the log will be too long and may confuse us.

**Review by aceforeverd — APPROVED** [review]

**Review by dl239 — APPROVED** [review]

**Pull request merged**